packages/form/src/react/vertical-layout.js

```diff
@@ -14,7 +14,7 @@ const VerticalLayout = (props, context) => (
     {React.Children.map(props.children, (child, i) => (
       <div {...styles.child(props)}>
         {React.cloneElement(child, {
-          style: { ...child.props.style, width: '100%' }
+          style: { ...child.props.style, width: '100%', maxWidth: 'none' }
```

Comment:
Note that this addition to Form.VerticalLayout is required now.  Makes sense to have been there before.  Now it's required to support this use case.